include/gazebo_geotagged_images_plugin.h

```diff
@@ -96,6 +99,7 @@ class GAZEBO_VISIBLE GeotaggedImagesPlugin : public SensorPlugin
     event::ConnectionPtr        _newFrameConnection;
     std::string                 _storageDir;
     ignition::math::Vector3d    _lastGpsPosition;
+    ignition::math::Angle       _hfov;      // Horizontal fov
```

Comment:
Can you please use Doxygen annotations for comments?
```suggestion
    ignition::math::Angle       _hfov;      ///< Horizontal fov
```